lbr/lbr.py

```diff
@@ -220,30 +220,48 @@ def mrn_cond(l):
       if C.any_in(vecs, ''.join(v2ii2v_srcs)): inc_stat('V2I transition-Penalty')
       elif C.any_in(vecs, v2ii2v_dst): inc_stat('I2V transition-Penalty')
     elif v2ii2v_dst: inc_stat('V2I transition-Penalty')
-  if size > 1:
```

Comment:
It seems your code needlessly change larger block of code.
For example, this check for size > 1 isn't there after you merge.
Please re-merge your new stats/logic on the latest repo